Tribler/Core/Modules/restapi/util.py

```diff
@@ -23,42 +26,84 @@ def return_handled_exception(request, exception):
     database in which case it is a tuple or a remote search result in which case it is a dictionary.
     """
     if isinstance(torrent, dict):
         return convert_remote_torrent_to_json(torrent)
-    return convert_db_torrent_to_json(torrent)
+    return convert_db_torrent_to_json(torrent, include_rel_score=True)
 
 
-def convert_db_channel_to_json(channel):
+def convert_db_channel_to_json(channel, include_rel_score=False):
     """
     This method converts a channel in the database to a JSON dictionary.
     """
-    return {"id": channel[0], "dispersy_cid": channel[1].encode('hex'), "name": channel[2], "description": channel[3],
-            "votes": channel[5], "torrents": channel[4], "spam": channel[6], "modified": channel[8],
-            "subscribed": (channel[7] == VOTE_SUBSCRIBE)}
+    res_json = {"id": channel[0], "dispersy_cid": channel[1].encode('hex'), "name": channel[2],
+                "description": channel[3], "votes": channel[5], "torrents": channel[4], "spam": channel[6],
+                "modified": channel[8], "subscribed": (channel[7] == VOTE_SUBSCRIBE)}
+
+    if include_rel_score:
+        res_json["relevance_score"] = channel[9]
 
+    return res_json
 
-def convert_db_torrent_to_json(torrent):
+
+def convert_db_torrent_to_json(torrent, include_rel_score=False):
     """
     This method converts a torrent in the database to a JSON dictionary.
     """
     torrent_name = torrent[2] if torrent[2] is not None else "Unnamed torrent"
 
-    return {"id": torrent[0], "infohash": torrent[1].encode('hex'), "name": torrent_name, "size": torrent[3],
-            "category": torrent[4], "num_seeders": torrent[5] or 0, "num_leechers": torrent[6] or 0,
-            "last_tracker_check": torrent[7] or 0}
+    res_json = {"id": torrent[0], "infohash": torrent[1].encode('hex'), "name": torrent_name, "size": torrent[3],
+                "category": torrent[4], "num_seeders": torrent[5] or 0, "num_leechers": torrent[6] or 0,
+                "last_tracker_check": torrent[7] or 0}
+
+    if include_rel_score:
+        res_json["relevance_score"] = torrent[9]
+
+    return res_json
 
 
 def convert_remote_torrent_to_json(torrent):
     """
     This method converts a torrent that has been received by remote peers in the network to a JSON dictionary.
     """
     torrent_name = torrent['name'] if torrent['name'] is not None else "Unnamed torrent"
+    relevance_score = relevance_score_remote_torrent(torrent_name)
 
     return {'id': torrent['torrent_id'], "infohash": torrent['infohash'].encode('hex'), "name": torrent_name,
             'size': torrent['length'], 'category': torrent['category'], 'num_seeders': torrent['num_seeders'],
-            'num_leechers': torrent['num_leechers'], 'last_tracker_check': 0}
+            'num_leechers': torrent['num_leechers'], 'last_tracker_check': 0,
+            'relevance_score': relevance_score}
+
+
```

Comment:
PEP8 specifies that there should be two newlines between methods outside a class ;)